### ISSUE TITLE
Error: Painter not active

### ISSUE THREAD

**Issue by Ben-Mack** [maggio-a/texture-defrag#3]
This is the result where I built and ran on Ubuntu 18.04 (using VirtualBox on Windows 10, because I haven't had successful build directly on Windows):

![image](https://user-images.githubusercontent.com/6309393/120325660-14c1b980-c312-11eb-88c4-a17c2c0abd89.png)

How can I solve this problem? 
Much appreciated!

**Comment by Zeeraktahir** [comment]
Same Issue here

![image](https://user-images.githubusercontent.com/112944026/203055732-5e310edc-18b3-4142-8262-ce09d40db79e.png)

**Comment by Mobeen-Invozone** [comment]
I am facing the same issue when doing texture defragmentation of my models. Can someone help with this issue????

![Screenshot 2022-11-23 153914](https://user-images.githubusercontent.com/111307115/203526517-77193961-d95c-44c3-aa34-ec78307deebb.png)

**Comment by shimon-vainer** [comment]
I think you have to load a mesh with existing texture for this to work.

**Comment by Mobeen-Invozone** [comment]
Yes, I am loading a .obj model which has an mtl and multiple textures but it throws this error not sure why.
